plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -73,13 +75,20 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
               labels: 'labels'
             };
 
+            const networkType = getNetworkType(
+              findNestedPropertyInObject(
+                appDefinition, 'container.docker.network'
+              ),
+              appDefinition
+            );
+
             const containerPortMappings = findNestedPropertyInObject(
               appDefinition, 'container.docker.portMappings'
             );
             if ((portDefinitions == null || portDefinitions.length === 0) &&
```

Comment:
That conditional causes a bug, that even when we have BRIDGE/USER network it still works like it's a HOST network. The thing is that Marathon will add `portDefinitinos` section with a `servicePort` therefore this conditional always will be true and you will see something like

![screen shot 2017-01-11 at 16 18 08](https://cloud.githubusercontent.com/assets/186223/21854378/d19df808-d81a-11e6-9980-6378954baa08.png)

I.e. take this JSON, launch an app and go to it's configuration tab and notice that even you don't see `portDefinitions` in the form it will be in the appDefinition you are getting from the server ¯\_(ツ)_/¯

so in my case I went for just checking
```javascript
if( containerPortMappings != null && containerPortMappings.length !== 0` ) {
...
}
```

```json
{
  "id": "/dima",
  "instances": 1,
  "cpus": 1,
  "mem": 128,
  "disk": 0,
  "gpus": 0,
  "backoffSeconds": 1,
  "backoffFactor": 1.15,
  "maxLaunchDelaySeconds": 3600,
  "container": {
    "type": "DOCKER",
    "docker": {
      "image": "nginx",
      "network": "BRIDGE",
      "portMappings": [
        {
          "name": "memeememme",
          "hostPort": 8091,
          "containerPort": 2020,
          "protocol": "tcp",
          "servicePort": 10001,
          "labels": {
            "VIP_0": "/dima:2020"
          }
        }
      ],
      "privileged": false,
      "forcePullImage": false
    }
  },
  "upgradeStrategy": {
    "minimumHealthCapacity": 1,
    "maximumOverCapacity": 1
  },
  "unreachableStrategy": {
    "inactiveAfterSeconds": 900,
    "expungeAfterSeconds": 604800
  },
  "killSelection": "YoungestFirst",
  "requirePorts": true
}
```